lib/bb-ruby.rb

```diff
@@ -118,7 +118,7 @@ module BBRuby
       /\[dd\](.*?)\[\/dd\]/mi,
       '<dd>\1</dd>',
       'Definition definitions',
-      '[dd]my definition[/dd',
+      '[dd]my definition[/dd]',
```

Comment:
Thanks for the above changes.  Can you make these corrections a separate commit from the new feature added below?  One commit for the "fixes" and one for the new feature?  Thanks